src/components/sdk/PreviewVideo/index.tsx

```diff
@@ -9,10 +9,11 @@ import { useVideoInputs } from '../../../providers/DevicesProvider';
 import { useLocalVideo } from '../../../providers/LocalVideoProvider';
 import { useLogger } from '../../../providers/LoggerProvider';
 import { useMeetingManager } from '../../../providers/MeetingProvider';
+import { styledComponentsConfig } from '../../../utils/style';
 import VideoTile from '../../ui/VideoTile';
 import { BaseSdkProps } from '../Base';
 
-const StyledPreview = styled(VideoTile)`
+const StyledPreview = styled(VideoTile).withConfig(styledComponentsConfig)`
```

Comment:
This is to handle a breaking change, in v5 the custom props such as `isEnabledForScreenSharing` is not forwarded to DOM element because it's not a native HTML attribute. So these non-native props are filtered. In v6 this filter is no longer provided by default, we follow the below guide to preserver the v5 behavior.
- https://styled-components.com/docs/faqs#shouldforwardprop-is-no-longer-provided-by-default